disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +208,13 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
```

Comment:
The  modules purpose reads "This module has a bunch of functions about waiting for an AWS resource to become available". The backoff function is doing exactly that. I think it is in the right place.